Document use of `opensafely codelists add` (#1713)
opensafely/documentation: docs/codelist-project.md

Your example research template doesn't include any codelists but the folder structure and text files that are needed to include codelists already exist.

> Take a look at the `codelists/codelists.txt` file in the repo, this file is currently empty but you can add one (or more) lines to this file that specify the codelists that you need for your research project.

Reviewer: ```suggestion
Take a look at the `codelists/codelists.txt` file in the repo, this file is currently empty but any codelists that you add to your project will appear here.
```

> If you want to add a codelist from [OpenCodelists](https://www.opencodelists.org) to your project you can edit this file manually (see below) or by using the `opensafely codelists add` command.

Reviewer: ```suggestion
You can add a codelist from [OpenCodelists](https://www.opencodelists.org) to your project by [manually editing](#manually-editing-codelists.txt) the codelists.txt file or by using the `opensafely codelists add` command.
```

For example, running the following command in your terminal:

```bash
opensafely codelists add https://www.opencodelists.org/codelist/opensafely/covid-identification/2020-06-03/
```

will add the [OpenSAFELY COVID Identification](https://www.opencodelists.org/codelist/opensafely/covid-identification) codelist to `codelists.txt` and also download and add `opensafely-covid-identification.csv` to your project.

> ### Editing codelists.txt

Reviewer: ```suggestion
### Manually editing codelists.txt
```
